templates/shared/forms/interactive/embedding.html

```diff
@@ -1,124 +1,249 @@
-            <label class="p-heading--5" for="about-use-case">Tell us more about your needs:</label>
-            <textarea id="about-use-case"></textarea>
+
+    <div class="p-section">
+      <div class="row--50-50">
+        <div class="col">
+          <h1 id="modal-title">Let's get in touch</h1>
+        </div>
+        <div class="col-5">
+          <p id="modal-description">
+            <!-- Canonical partners with silicon companies, board manufacturers and ODMs to help the industry bring smart device and software capabilities to market faster. Tell us about your project so we can bring the right Canonical team members to the conversation to help you. -->
+          </p>
+        </div>
+      </div>
+    </div>
+    <form action="/marketo/submit" method="post" id="mktoForm_%% formid %%">
+      <div class="p-section">
+        <hr class="p-rule is-fixed-width" />
+        <div class="row--50-50 js-formfield">
+          <div class="col">
+            <label for="canonicalProducts" class="p-heading--4 js-formfield-title">
+              Does your product or service include Canonical products?
+            </label>
+          </div>
+          <div class="col">
+            <fieldset class="p-fieldset-section js-remove-radio-names">
+              <label class="p-radio">
+                <input class="p-radio__input"
+                       required
+                       type="radio"
+                       aria-labelledby="canonicalProductsYes"
+                       name="canonicalProducts"
+                       value="Yes" />
+                <span class="p-radio__label" id="canonicalProductsYes">Yes</span>
+              </label>
+              <label class="p-radio">
+                <input class="p-radio__input"
+                       required
+                       type="radio"
+                       aria-labelledby="canonicalProductsNo"
+                       name="canonicalProducts"
+                       value="No" />
+                <span class="p-radio__label" id="canonicalProductsNo">No</span>
+              </label>
+            </fieldset>
+          </div>
+        </div>
+      </div>
+      <div class="p-section">
+        <hr class="p-rule is-fixed-width" />
+        <div class="row--50-50 js-formfield">
+          <div class="col">
+            <label for="about-use-case" class="p-heading--4 js-formfield-title">Tell us more about your needs</label>
+          </div>
+          <div class="col">
+            <textarea id="about-use-case" rows="5"></textarea>
           </div>
         </div>
-        <h3 class="p-heading--5">How should we get in touch?</h3>
-        <div class="row u-no-padding">
-          <div class="col-6">
-            <form action="/marketo/submit" onsubmit="canonicalProducts()" method="post" id="mktoForm_%% formid %%" class="modal-form">
+      </div>
+      <div class="p-section">
+        <hr class="p-rule is-fixed-width" />
+        <fieldset class="p-fieldset-section js-formfield"
```

Comment:
```suggestion
        <fieldset class="p-fieldset-section"
```
This is causing this values to be added to the 'comments_from_lead' textarea. But as these are attached directly to the payload it isn't needed